### PR TITLE
Add simple menu to console application

### PR DESCRIPTION
**Pull request by peter-csala**
# Description
- Added a Splash screen with a Polly logo ASCII art
- Added a simple menu to ease the usage of the console application

# Preview
Polly Demo Console with Menu

### CODE REVIEW

File: PollyTestClientConsole/Menu/ConsoleMenu.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
+namespace PollyTestClientConsole.Menu
```
Comment (martincostello) [inline PollyTestClientConsole/Menu/ConsoleMenu.cs:1]: File-scoped namespaces in the new files please.

File: PollyTestClientConsole/Menu/ConsoleMenu.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    {
+                        Console.Clear();
+                        items[index].Handler();
+                        Console.ReadKey();
+                        Console.WriteLine();
+                        Console.WriteLine("Press any key to return to menu");
+                        Console.ReadKey();
+                        WriteMenu(items, items[index]);
+                    }
+                    ,
```
Comment (martincostello) [inline PollyTestClientConsole/Menu/ConsoleMenu.cs:73]:
```suggestion
                    },
```

File: PollyTestClientConsole/Program.cs [hunk 3]
```diff
@@ -20,72 +20,101 @@ private static async Task Main()
...
-            // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-            // Concurrency Limiter demos
-            // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-            //asyncDemo = new AsyncDemo10_SharedConcurrencyLimiter();
-            //asyncDemo = new AsyncDemo11_MultipleConcurrencyLimiters();
-
-            await asyncDemo!.ExecuteAsync(cancellationToken, progress);
-        }
+            new("Sync: No strategy",
```
Comment (martincostello) [inline PollyTestClientConsole/Program.cs:29]: Can we put the async ones first please?

Not only would I have thought async be preferred to sync in general, but also they're first alphabetically.

File: PollyTestClientConsole/README.md [hunk 4]
```diff
@@ -17,17 +17,16 @@ flowchart LR
 
 ## Exposed functionality
 
-- It lets you start and stop any demo.
+- It lets you start, stop and re-run any demo.
 - While the demo is running it prompts the logs.
 - To stop the demo press _any_ key.
 - When you stop the demo you can scrutinize the logs and check the statistics.
-- To exit from the application press _any_ key after you assessed the prompted messages.
+- To go back to the menu press _any_ key after you assessed the prompted messages.
```
Comment (martincostello) [inline PollyTestClientConsole/README.md:24]: image

Comment (peter-csala) [reply PollyTestClientConsole/README.md:24]: Shall I change here anything?

Comment (martincostello) [reply PollyTestClientConsole/README.md:24]: No :)

**Review by martincostello — APPROVED** [review]

**Pull request merged**